kivy/uix/textinput.py

```diff
@@ -454,7 +454,7 @@ class TextInput(FocusBehavior, Widget):
                   'on_quad_touch')
 
     def __init__(self, **kwargs):
-        self.always_release_bubble = kwargs.get('always_release_bubble', False)
+        self.always_release_bubble = kwargs.get('always_release_bubble', True)
```

Comment:
But u said that there should be a way so that users can set always_release for the BubbleButton object. From here https://github.com/kivy/kivy/blob/master/kivy/uix/behaviors/button.py#L168-L169 if self.dispatch('on_release') event has to be triggered then either always_release should be True or self.collide_point(*touch.pos) should  be True. 
I created a new variable which can be set by the users .Other way would be to change the touch.pos in local or windows co-ordinates.(not sure about the second option)